packages/hooks-react/src/useBillingPortalURL.tsx

```diff
@@ -0,0 +1,14 @@
+import { useMutation } from 'react-query';
+import { getModule } from '@jwp/ott-common/src/modules/container';
+import CheckoutController from '@jwp/ott-common/src/controllers/CheckoutController';
+
+const useBillingPortalURL = () => {
+  const checkoutController = getModule(CheckoutController);
+
+  return useMutation({
+    mutationKey: 'billing-portal',
+    mutationFn: () => checkoutController.generateBillingPortalUrl(window.location.href),
+  });
+};
```

Comment:
The current URL should be forwarded (it still used `window.location`):

```tsx
const useBillingPortalURL = (currentUrl: string) => {
  // ...
 mutationFn: () => checkoutController.generateBillingPortalUrl(currentUrl),
```

Reply:
Done. I did it little bit differently, but I think you'll approve of it.